### ISSUE TITLE
x509: failed to load system roots and no roots provided

### ISSUE THREAD

**Issue by rimey** [kubernetes/heapster#444]
I'm following the instructions in heapster/docs/google.md and seeing heapster crash-loop with the following logs:

```
I0731 11:50:56.696529       1 heapster.go:55] /heapster --source=kubernetes:'' --sink=gcm --sink=gcl --poll_duration=2m --stats_resolution=1m
I0731 11:50:56.696634       1 heapster.go:56] Heapster version 0.17.0
I0731 11:50:56.697922       1 kube_factory.go:168] Using Kubernetes client with master "https://10.147.240.1:443" and version "v1"
I0731 11:50:56.697957       1 kube_factory.go:169] Using kubelet port 10255
E0731 11:50:59.720275       1 core.go:118] [GCM] list metrics failed Get https://www.googleapis.com/cloudmonitoring/v2beta2/projects/gke-playground/metricDescriptors: x509: failed to load system roots and no roots provided
I0731 11:50:59.720377       1 driver.go:113] created GCM sink
F0731 11:50:59.720420       1 heapster.go:62] Get https://www.googleapis.com/cloudmonitoring/v2beta2/projects/gke-playground/metricDescriptors: x509: failed to load system roots and no roots provided
```

Despite #382, there's no ca-certificates.crt in the v0.17.0 image. Could that be the problem?

```
$ docker run --rm --entrypoint=/bin/cat kubernetes/heapster:v0.10.0 /etc/ssl/certs/ca-certificates.crt
/bin/cat: /etc/ssl/certs/ca-certificates.crt: No such file or directory
```

**Comment by vishh** [comment]
@mikedanese: Can you take a look at this issue? Will heapster docker images have to be built on a GCE machine in-order to avoid this issue?

**Comment by Lswith** [comment]
the same thing is occurring in v0.16.0 as well.

**Comment by mikedanese** [comment]
Should be fixed by https://github.com/GoogleCloudPlatform/heapster/pull/445?

**Comment by piosz** [comment]
I assume it was fixed by  #445. Please use a stable version of Heapster and reopen in case of problems.
